module/Database/src/Model/SubDecision/OrganRegulation.php

```diff
@@ -0,0 +1,252 @@
+     */
+    #[Column(type: 'date')]
+    protected DateTime $date;
+
+    /**
+     * If the regulation was approved.
+     */
+    #[Column(type: 'boolean')]
+    protected bool $approval;
+
+    /**
+     * If there were changes made.
+     */
+    #[Column(type: 'boolean')]
+    protected bool $changes;
+
+    /**
+     * Get the author.
+     */
+    public function getAuthor(): ?Member
+    {
+        return $this->author;
+    }
+
+    /**
+     * Set the author.
+     */
+    public function setAuthor(Member $author): void
+    {
+        $this->author = $author;
+    }
+
+    /**
+     * Get the type.
+     */
+    public function getOrganType(): OrganTypes
+    {
+        return $this->organType;
+    }
+
+    /**
+     * Set the organ type
+     */
+    public function setOrganType(OrganTypes $organType): void
+    {
+        $this->organType = $organType;
+    }
+
+    /**
+     * Get the name.
+     */
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    /**
+     * Set the name.
+     */
+    public function setName(string $name): void
+    {
+        $this->name = $name;
+    }
+
+    /**
+     * Get the version.
+     */
+    public function getVersion(): string
+    {
+        return $this->version;
+    }
+
+    /**
+     * Set the version.
+     */
+    public function setVersion(string $version): void
+    {
+        $this->version = $version;
+    }
+
+    /**
+     * Get the date.
+     */
+    public function getDate(): DateTime
+    {
+        return $this->date;
+    }
+
+    /**
+     * Set the date.
+     */
+    public function setDate(DateTime $date): void
+    {
+        $this->date = $date;
+    }
+
+    /**
+     * Get approval status.
+     */
+    public function getApproval(): bool
+    {
+        return $this->approval;
+    }
+
+    /**
+     * Set approval status.
+     */
+    public function setApproval(bool $approval): void
+    {
+        $this->approval = $approval;
+    }
+
+    /**
+     * Get if changes were made.
+     */
+    public function getChanges(): bool
+    {
+        return $this->changes;
+    }
+
+    /**
+     * Set if changes were made.
+     */
+    public function setChanges(bool $changes): void
+    {
+        $this->changes = $changes;
+    }
+
+    /**
+     * Get the content.
+     */
+    public function getContent(): string
+    {
+        $template = $this->getTemplate();
+        $template = str_replace('%NAME%', $this->getName(), $template);
+        if (null === $this->getAuthor()) {
+            $template = str_replace('%AUTHOR%', 'onbekend', $template);
+        } else {
+            $template = str_replace('%AUTHOR%', $this->getAuthor()->getFullName(), $template);
+        }
+
+        if (OrganTypes::Committee === $this->getOrganType()) {
+            $template = str_replace('%TYPE%', 'commissie', $template);
+        } else {
+            $template = str_replace('%TYPE%', 'dispuuts', $template);
+        }
```

Comment:
I would suggest changing this, because contrary what the internal regulations say, the board insists on having committee regulations for the financial audit committee. In this case, those would be "dispuutsreglement"?

Or, my preference, we don't implement financial audit committees here, don't offer this as a choice to the user and throw an error when it is not one of the two supproted types

Reply:
All options are shown, but only `committee` and `fraternity` are valid options:

![image](https://github.com/GEWIS/gewisdb/assets/4983571/747d5874-00ea-4780-9da4-128f3fba3253)

In the event the board really wants to have organ regulations for other organ types, such as the finanical audit committee, they should first change the IR imho.